src/LandIce/evaluators/LandIce_PressureMeltingEnthalpy_Def.hpp

```diff
@@ -51,6 +51,17 @@ void PressureMeltingEnthalpy<EvalT,Traits,PressST,SurfTempST>::
 postRegistrationSetup(typename Traits::SetupData d, PHX::FieldManager<Traits>& fm)
 {}
 
+template<typename EvalT, typename Traits, typename PressST, typename SurfTempST>
+void PressureMeltingEnthalpy<EvalT,Traits,PressST,SurfTempST>::
+preEvaluate(typename Traits::EvalData d)
+{
+  const double powm6 = 1e-6; // [k^2], k=1000
+
+  for (std::size_t cell = 0; cell < d.numCells; ++cell)
+    for (std::size_t node = 0; node < numNodes; ++node)
+      surfaceEnthalpy(cell,node) = rho_i * c_i * ( std::min(surfaceTemp(cell,node),Tm) - T0 ) * powm6;
```

Comment:
Are you computing this to use it somewhere else in volume FM `preEvaluate`? I don't see it but maybe it's not in this PR or you haven't implemented it yet.

Reply:
This is scattered (with `PHAL::ScatterScalarNodalParameter`) and then used in the Dirichlet conditions.

Reply:
okay thanks